index.bs

```diff
@@ -3491,15 +3512,40 @@ and at least one of the following three conditions is true:
             and it is not the case that both are [=callback interfaces=].
         1.  The interface type is not a [=callback interface=].
     </ol>
-1.  One type is a [=union type=] or nullable union type,
-    the other is neither a union type nor a nullable union type, and each
-    [=member type=] of the first is distinguishable
-    with the second.
-1.  Both types are either a union type or nullable union type, and each member type of the one
-    is distinguishable with each member type of the other.
-
-Note: [=Promise types=] do not appear in the above table, and as a consequence
-are not distinguishable with any other type.
+
+    <div class="example" id="example-distinguishability">
```

Comment:
https://heycam.github.io/webidl/#example-e7863d09 has a similar triple of examples, but I can split this one up.